manifests/repo/download_postgresql_org.pp

```diff
@@ -16,15 +16,21 @@
     $label1 = 'redhat'
     $label2 = 'rhel'
   }
+  $default_baseurl = "https://download.postgresql.org/pub/repos/yum/${postgresql::repo::version}/${label1}/${label2}-\$releasever-\$basearch"
```

Comment:
Hmmm, the tricky bit here will be that a user will have to have a static URL, or their own way of templating it on their own, if this is passed as a string way back in globals ... I guess I'm just not sure if one should allow the full static URL, or ask for a prefix, and we apply these late URL path bits (I mean the label1, label2 magic URL generation). I'm not that sure about how PostgreSQL recommends to mirror, it might be they say preserve their URL's ... or are less specific.

I'm really okay with this as a current strawman and probably would be fine enough to see it merged as it is (more because doing it with prefix only, is less flexible ultimately), just wondering if this has been considered.